demo/firebase-idtoken-flow-example/src/App.tsx

```diff
@@ -0,0 +1,528 @@
+    }
+    
+    let factorKey = await securityQuestion.recoverSecurityQuestionFactor(coreKitInstance, answer);
+    setBackupFactorKey(factorKey);
+    uiConsole("Security Question share: ", factorKey);
+  }
+
+  const logout = async () => {
+    if (!coreKitInstance) {
+      throw new Error("coreKitInstance not found");
+    }
+    await coreKitInstance.logout();
+    setCoreKitStatus(COREKIT_STATUS.NOT_INITIALIZED);
+    uiConsole("Log out");
+    setProvider(undefined);
+  };
+
+  const getUserInfo = (): void => {
+    const user = coreKitInstance?.getUserInfo();
+    uiConsole(user);
+  };
+
+  const exportShare = async (): Promise<void> => {
+    if (!coreKitInstance) {
+      throw new Error("coreKitInstance is not set");
+    }
+    uiConsole("export share type: ", exportTssShareType);
+    const factorKey = generateFactorKey();
+    await coreKitInstance.createFactor({
+      shareType: exportTssShareType,
+      factorKey: factorKey.private
+    });
+    uiConsole("Export factor key: ", factorKey);
+  }
+
+  const deleteFactor = async (): Promise<void> => {
+    if (!coreKitInstance) {
+      throw new Error("coreKitInstance is not set");
+    }
+    const pubBuffer = Buffer.from(factorPubToDelete, 'hex');
+    const pub = Point.fromBufferSEC1(pubBuffer);
+    await coreKitInstance.deleteFactor(pub.toTkeyPoint());
+    uiConsole("factor deleted");
+  }
+
+  const getChainID = async () => {
+    if (!web3) {
+      uiConsole("web3 not initialized yet");
+      return;
+    }
+    const chainId = await web3.eth.getChainId();
+    uiConsole(chainId);
+    return chainId;
+  };
+
+  const getAccounts = async () => {
+    if (!web3) {
+      uiConsole("web3 not initialized yet");
+      return;
+    }
+    const address = (await web3.eth.getAccounts())[0];
+    uiConsole(address);
+    return address;
+  };
+
+  const getBalance = async () => {
+    if (!web3) {
+      uiConsole("web3 not initialized yet");
+      return;
+    }
+    const address = (await web3.eth.getAccounts())[0];
+    const balance = web3.utils.fromWei(
+      await web3.eth.getBalance(address) // Balance is in wei
+    );
+    uiConsole(balance);
+    return balance;
+  };
+
+  const signMessage = async (): Promise<any> => {
+    if (!web3) {
+      uiConsole("web3 not initialized yet");
+      return;
+    }
+    const fromAddress = (await web3.eth.getAccounts())[0];
+    const originalMessage = [
+      {
+        type: "string",
+        name: "fullName",
+        value: "Satoshi Nakamoto",
+      },
+      {
+        type: "uint32",
+        name: "userId",
+        value: "1212",
+      },
+    ];
+    const params = [originalMessage, fromAddress];
+    const method = "eth_signTypedData";
+    const signedMessage = await (web3.currentProvider as any)?.sendAsync({
+      id: 1,
+      method,
+      params,
+      fromAddress,
+    });
+    uiConsole(signedMessage);
+  };
+
+  const resetAccount = async (): Promise<void> => {
```

Comment:
we should mark this function as dangerous and tell app to not use it in production, should be use only for development or best is to disable this function for mainnet.

Reply:
this is not a function anymore directly in the SDK, we have this in the example app. We can mark it as dangerous there.

Reply:
yes still in example, leave a comment that if this function is called we wont be responsible for recovering account. So ideally use it only for development

Reply:
add a check in example, if network is mainnet and they are calling this function, throw error.

Reply:
also show a note in the UI of example, this is very critical

Reply:
done